packages/shared/src/components/tooltips/notifications/NotificationPreferenceMenu.tsx

```diff
@@ -67,36 +68,31 @@ export const NotificationPreferenceMenu = ({
     return <NotifIcon />;
   };
 
-  const Copy = (): ReactElement => {
+  const label = useMemo((): string => {
     if (!notification) {
       return null;
     }
 
     if (isFetching) {
-      return <>Fetching your preference</>;
+      return 'Fetching your preference';
     }
 
     const isMuted =
       preferences[0]?.status === NotificationPreferenceStatus.Muted;
     const copy = notificationMutingCopy[notification?.type];
 
-    return <>{isMuted ? copy.unmute : copy.mute}</>;
-  };
+    return isMuted ? copy.unmute : copy.mute;
+  }, [notification, preferences, isFetching]);
 
   return (
-    <PortalMenu
+    <ContextMenu
       disableBoundariesCheck
       id={contextId}
       className="menu-primary"
       animation="fade"
       onHidden={onClose}
-    >
-      <Item className="w-64 py-1 typo-callout" onClick={onItemClick}>
-        <span className="flex w-full flex-row items-center gap-1">
-          <Icon />
-          <Copy />
```

Comment:
This seem to have 2 icons, why only one now?

Reply:
Copy was not an icon here, but rather a variable for the label, the naming was a bit confusing, although 'copy' is a valid name, I initially confused it for an icon too, so I renamed to 'label' as seen above 🙏